guides/plugins/apps/platform-sh-template.md

```diff
@@ -0,0 +1,208 @@
+
+An example for the `manifest.xml` would be:
+
+```xml
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/app-system/0.1.0/src/Core/Content/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>myAppName</name>
+    </meta>
+    <setup>
+        <secret>myAppSecret</secret>
+    </setup>
+</manifest>
+```
+
+An example for the [.platform.app.yaml](https://github.com/shopwareLabs/AppTemplate/blob/master/.platform.app.yaml) would be:
+```yaml
+variables:
+    env:
+        APP_NAME: myExampleApp
+        APP_SECRET: myAppSecret
+```
+
+These should also be changed in the `.env` to develop locally.
+
+## Create the manifest.xml
+
+To easily create `manifest.xml`-files you can use the `bin/console app:create-manifest` command. This command will generate a manifest from the [template](https://github.com/shopwareLabs/AppTemplate/blob/master/templates/manifest-template.xml). For testing purposes this is the default manifest template from our example app.  
+In there you can use `{{ APP_NAME }}`,`{{ APP_SECRET }}`,`{{ APP_URL_CLIENT }}` and `{{ APP_URL_BACKEND }}` which will get replaced by the values configured in your `.env`. Further more you can use your own variables like `{{ MY_OWN_VARIABLE }}` and declare them when executing the command like this `bin/console app:create-manifest MY_OWN_VARIABLE=MY_OWN_VARIABLE_VALUE`.  
+This allows you to easily change URL's in your `manifest.xml` without changing the `.env` file. The generated `manifest.xml` can be found in `build/dev`.  
+
+## Development
+
+This is a symfony based development template. To register your app you only need to configure your manifest. The registration URL is `https://www.my-app.com/registration`.
+
+The `SwagAppsystem\Client` and `SwagAppsystem\Event` will be injected when you need them.
+For example: 
+
+```php
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\SwagAppsystem\Client;
+use App\SwagAppsystem\Event;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+class OrderController
+{
+    /**
+     * @Route("/order/placed/event", methods={"POST"})
+     */
+    public function orderPlacedEvent(Client $client, Event $event): Response
+    {
+        //nth
+    }
+}
+```
+
+## Testing
+
+To test your app you can use [PHPUnit](https://phpunit.de/index.html)  
+You can write your own tests in `tests`  
+and execute them executing `vendor/bin/phpunit`
+
+To check your codestyle you can use [EasyCodingStandard](https://github.com/symplify/easy-coding-standard)  
+Just execute `vendor/bin/ecs check` to check your code or add `--fix` to also fix your code.
+
+## Code quality
+
+To improve your code style we added [EasyCodingStandard](https://github.com/symplify/easy-coding-standard) and for testing purposes [PHPUnit](https://phpunit.de/index.html).
+
```

Comment:
Why do you explain them under "Code quality", even more because you explained PHPunit and ecs before.

I'd suggest, you just remove this.
```suggestion
```